### PR TITLE
Fix #3453: Merge admin_auth_activity.xml into one file

### LINKED ISSUE #3453: Merge admin_auth_activity.xml into single xml
Currently there are 4 versions of `admin_auth_activity.xml` file, merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by addy-zero**
## Explanation

Fixes #3453 by merging 4 different layout files into one

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

| Before  | After |
| ------------- | ------------- |
|   |   |
|   |    |
|   |   |
|   |   |

### CODE REVIEW

**Review by vrajdesai78 — APPROVED** [review]

File: app/src/main/res/values/dimens.xml
Comment (rt4914) [inline app/src/main/res/values/dimens.xml]: Update to `<!-- AdminAuthActivity -->`

File: app/src/main/res/values/dimens.xml
Comment (rt4914) [inline app/src/main/res/values/dimens.xml]: At top of this file create a value 
`<dimen name="match_parent">-1px</dimen>`
and then use that `match_parent` here.

**Comment by rt4914** [comment]
@adizcode I have informed @vrajdesai78 to schedule a meeting today (all three of us) so that I can explain you what changes are expected in this PR. You are on right track but still we need some changes to make it better.

**Comment by addy-zero** [comment]
@rt4914 Thank you. Happy to learn from you all. 

Just a tiny request, can we move the meet to tomorrow since I'm already occupied today?

**Comment by addy-zero** [comment]
@rt4914 I've made the requested changes and updated the screenshots. The values were taken directly from the mockups.

The only visual difference lies in the mobile-landscape setting. The reason for that is that the **previous XML layout used a fixed margin for the TextInputLayout**, when in fact we are supposed to be using a variable margin with fixed content (if I'm not wrong).

I'm yet to pull changes from develop. Will do that if everything looks good to you.

**Comment by rt4914** [comment]
@adizcode Will review this today. If you end up reading this message before my review, please merge it with latest develop, else its fine.

**Comment by addy-zero** [comment]
@rt4914 I had merged the new changes, PTAL.

File: app/src/main/res/values-land/dimens.xml
Comment (rt4914) [inline app/src/main/res/values-land/dimens.xml]: Change to `<!-- AdminAuthActivity -->` here and in other files too.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@adizcode Just update the screenshots as per latest code. Otherwise everything looks great.


FYI: This will take time in merging as I will have to explain to Ben about the -1px and -2px values as a reminder and get his final approval as this is something which we will apply to all other files in time. That being said you can work on other issues in that time.

**Comment by addy-zero** [comment]
@rt4914 Done and updated the screenshots. Please merge when you can :)

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by rt4914** [comment]
@adizcode Just merge this with latest develop and its ready to merge.

**Comment by addy-zero** [comment]
@rt4914 One of the checks is apparently failing. However, my branch seems to work locally.

Edit: It looks like a Gradle crash. Re-running the tests should pass all the checks.
Re-edit: All checks passing!

**Comment by bhaktideshmukh** [comment]
@adizcode 
Your branch has conflicts , try resolving them
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github
Thanks

**Comment by addy-zero** [comment]
@rt4914 It seems that the layout of admin_auth_activity.xml was changed by someone else's PR. All the four files as well as their particular dimens files were modified.

However, since I had already merged all the four files into one, how can I proceed further and resolve these conflicts?

**Pull request closed without merging**
